pybossa/api/__init__.py

```diff
@@ -80,6 +80,10 @@
 from pybossa.auth.task import TaskAuth
 from pybossa.service_validators import ServiceValidators
 import requests
+from sqlalchemy.sql import text
+from pybossa.model.user import User
```

Comment:
unused?

Reply:
I use both of these in the method task_prgress

Reply:
That's right. I will remove it